.github/actions/install-build-tools/action.yml

```diff
@@ -6,6 +6,6 @@ runs:
   - name: Install Build Tools
     shell: bash
     run: |
-      export BUILD_TOOLS_SHA=47d4bb016f47d89938898c794db80b9f98d78ad7
+      export BUILD_TOOLS_SHA=1357329348b4b63f248bcf1a8d23023b43ae9336
```

Comment:
Let's make this `d5b87591842be19058e8d75d2c5b7f1fabe9f450` to catch the latest on `main` after your PR is merged (also catches my init fix)